src/pages/Login/index.js

```diff
@@ -42,11 +45,34 @@ function Login(props) {
     // dispatch(userActions.onUserUpdate(referrer));
   }
 
+  useEffect(() => {
+    dispatch(pathwayActions.getPathways());
+  }, [dispatch]);
+
+  const classes = useStyles();
+  // const isActive = useMediaQuery("(max-width:600px)");
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
+
   const onGoogleLoginFail = (errorResponse) => {
     // eslint-disable-next-line no-console
     console.log("onGoogle login fail", errorResponse);
   };
 
+  let pythonPathwayId;
+  pathway.data &&
+    pathway.data.pathways.forEach((pathway) => {
+      if (pathway.code === "PRGPYT") pythonPathwayId = pathway.id;
+    });
```

Comment:
Pathway code of Python (PRGPYT) should also probably be stored in a constant. 
```suggestion
  const pythonPathway = pathway.data &&
      pathway.data.pathways.find((pathway) => pathway.code === "PRGPYT");
  const pythonPathwayId = pythonPathway && pythonPathway.id;
```